modules/nodes-pods-vertical-autoscaler-configuring.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-vertical-autoscaler.adoc
+
+[id="nodes-pods-vertical-autoscaler-configuring_{context}"]
+= Using the Vertical Pod Autoscaler
+
+You can use the Vertical Pod Autoscaler (VPA) by creating a Vertical Pod Autoscaler Custom Resource (CR). The CR determines which Deployment or ReplicaSet it should analyze and determines the actions the VPA should take with the Pods associated with that Deployment or ReplicaSet. 
+
+.Procedure
+
+To create a VPA Custom Resource for a specific controller, either a Deployment or ReplicaSet:
+
+. Change to the project where the Deployment or ReplicaSet you want to autoscale is located.
+
+.. Create a VPA object YAML file:
++
+[source, yaml]
+----
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: vpa-recommender
+spec:
+  targetRef:
+    apiVersion: "apps/v1"
+    kind:       Deployment <1>
+    name:       frontend <2>
+  updatePolicy:
+    updateMode: "Auto" <3>
```

Comment:
@joelsmith  Do we get `updateMode` supported? I did no see this in the demo

Reply:
Yes, it is supported. I left it out of the demo because it is short, but we should probably have some test cases to verify that it works. Thanks for covering it in the docs @mburke5678